### PR TITLE
Add data file for geocat-comp climatology example

### PR DESCRIPTION
**Pull request by hCraker**
Added a subset of CESM output data which is to be used in a GeoCAT-comp example script

### CODE REVIEW

**Comment by anissa111** [comment]
Hey Heather, you'll also need to rebuild the registry file and add it to this PR! You should just need to run `create_registry.py` in your local to rebuild it.

**Comment by hCraker** [comment]
Thanks @anissa111 I forgot about that part. We should be good to go now!

File: registry.txt [hunk 1]
```diff
@@ -118,6 +118,7 @@ hdf_files/MLS-Aura_L2GP-IWC_v02-21-c02_2007d210.he5 30216d1ebe43c6fc32b662f2baf6
 hdf_files/MOD04_L2.A2001066.0000.004.2003078090622.he2 223905b8aba95cc7b11d72de7d17c22de18174547de765f1327021c1d100596a
 hdf_files/MOD05_L2.A2002161.1830.004.2003221153410.hdf 321ca849140cbe2a0284deb8a0b5b9d02bce23a42872f21fd1ed2dc9b19b9521
 hdf_files/avhrr.hdf 1d5cda404844b45b259001bb35b5f17167be0b331d608ad99f745b3f1053fde4
+netcdf_files/.DS_Store 64d520646ab8827f4165f1b257ca2e040e4f0b31865aeefee3521f8b0f1276ca
```
Comment (anissa111) [inline registry.txt:121]: It looks like something extra got pulled into the registry

Comment (hCraker) [reply registry.txt]: @anissa111 Thanks for catching this. I removed that line from the registry. Please let me know if anything else seems amiss.

**Review by anissa111 — APPROVED** [review]

**Pull request merged**